prow/external-plugins/cherrypicker/README.md

```diff
@@ -0,0 +1,31 @@
+# Cherrypicker
+
+**Note: This is a fork of [cherrypicker from kubernetes/test-infra](https://github.com/kubernetes/test-infra/tree/master/prow/external-plugins/cherrypicker) repository.
```

Comment:
Please add a note on which which commit this fork is based. Makes things easier in the long-run :)